src/app/app-routing.module.ts

```diff
@@ -3,13 +3,16 @@ import { Routes, RouterModule } from '@angular/router';
 import { LoginComponent } from './components/login/login.component';
 import { RegisterComponent } from './components/register/register.component';
 import { PerformerComponent } from './components/performer/performer.component';
+import { EmailconfirmationComponent } from './components/emailconfirmation/emailconfirmation.component';
 
 
 const routes: Routes = [
   { path: '',   redirectTo: '/login', pathMatch: 'full' },
+  { path: 'emailConfirmation',   redirectTo: '/emailConfirmation', pathMatch: 'full' },
```

Comment:
Wofür benötigst du diese Zeile?

Reply:
Rausgenommen, ich wollte ursprünglich damit verhindern, dass in der Adresszeile die Übergabeparameter angezeigt werden. Funktioniert aber scheinbar dann nicht mehr.